romancal/datamodels/schemas/reference_files/keyword_gainfact.schema

```diff
@@ -0,0 +1,12 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/asdf/asdf-schema-1.0.0"
+id: "http://stsci.edu/schemas/roman_datamodel/reference_files/keyword_gainfact.schema"
+type: object
+properties:
+  meta:
+    type: object
+    properties:
+      gain_factor:
```

Comment:
Does it make sense for this value to be negative?  Could add:

```yaml
gain_factor:
  title: Gain scale factor
  type: number
  minimum: 0.0
```
or if zero is also not permitted:
```yaml
gain_factor:
  title: Gain scale factor
  type: number
  minimum: 0.0
  exclusiveMinimum: true
```